core/lib/snapshots_applier/src/lib.rs

```diff
@@ -588,18 +588,27 @@ impl<'a> SnapshotsApplier<'a> {
             factory_deps.factory_deps.len()
         );
 
-        let all_deps_hashmap: HashMap<H256, Vec<u8>> = factory_deps
-            .factory_deps
-            .into_iter()
-            .map(|dep| (hash_bytecode(&dep.bytecode.0), dep.bytecode.0))
-            .collect();
-        storage
-            .factory_deps_dal()
-            .insert_factory_deps(
-                self.applied_snapshot_status.l2_block_number,
-                &all_deps_hashmap,
-            )
-            .await?;
+        // we cannot insert all factory deps because of field size limit triggered by UNNEST
```

Comment:
Nit: You can enclose "UNNEST" in backticks (i.e., format it as code) so that it won't trigger a spellcheck error.

Reply:
thanks, will use this trick next time